README.adoc

```diff
@@ -2311,6 +2311,8 @@ Run the following command to set up port forwarding to access the `inventory` mi
 kubectl port-forward svc/inventory-deployment 9443
 ```
 
+The `port-forward` command should hold up the command-line session.
```

Comment:
what does "hold up" mean in this context. Also, we should avoid using "should" to refer to a function as it implies uncertainty.

Reply:
- the word "should" can be removed
-"hold up" means that the command session cannot be typed in anything more. Any better word?